tasks/setup.py

```diff
@@ -498,6 +498,9 @@ def debug_keys(ctx: Context, config_path: Optional[str] = None):
         # https://github.com/pulumi/pulumi-command/blob/58dda0317f72920537b3a0c9613ce5fed0610533/provider/pkg/provider/remote/connection.go#L105-L118
         error("Private key is not provided in the config. Pulumi does not support Windows SSH agent.")
         info("Configure privateKeyPath and provide the privateKeyPassword if the key is encrypted.")
+    if not awsConf.privateKeyPath:
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
If not in there yet, could be part of the `setup` task